src/Raven.Server/RavenServer.cs

```diff
@@ -1352,6 +1357,8 @@ await ServerStore.SendToLeaderAsync(new InstallUpdatedServerCertificateCommand(C
                     AlertType.Certificates_ReplaceError,
                     NotificationSeverity.Error,
                     details: new ExceptionDetails(e)));
+
+                throw;
```

Comment:
where this method is used? this will not kill the server?

Reply:
all callers are wrapped with try/catch